scripts/rdp-enum-encryption.nse

```diff
@@ -96,9 +93,27 @@ local function enum_protocols(host, port)
       -- we can't tell if the protocol is accepted or not.
       table.insert(res_proto, ("%s: Unknown"):format(k))
     end
+
+    -- For servers that require TLS or NLA the only way to get the RDP protocol
+    -- version to negotiate TLS or NLA. This section does that for TLS. There
+    -- is no NLA currently.
+    if status and (v == 1) then
+      status, err = comm.socket:reconnect_ssl()
```

Comment:
`err` needs to be declared local, too. I find things like this with my Lua check script from [the Code Standards page on SecWiki](https://secwiki.org/w/Nmap/Code_Standards#Tools_to_help).

Reply:
I use `luacheck` but forgot to do so after refactoring.